code/run_capsule.py

```diff
@@ -5,60 +5,59 @@
 from datetime import datetime as dt
 from multiprocessing.pool import Pool
 from pathlib import Path
-from typing import Union
+from typing import Tuple, Union
 
 import h5py
 import matplotlib.pyplot as plt
 import numpy as np
 import seaborn as sns
-from aind_data_schema.core.processing import DataProcess, PipelineProcess, Processing, ProcessName
+from aind_data_schema.core.processing import DataProcess, ProcessName
+from aind_log_utils.log import setup_logging
 from oasis.functions import deconvolve
 from oasis.oasis_methods import oasisAR1, oasisAR1_f32, oasisAR2
 
 
-def write_output_metadata(
+def write_data_process(
     metadata: dict,
-    process_json_dir: str,
-    process_name: str,
     input_fp: Union[str, Path],
     output_fp: Union[str, Path],
-    start_date_time: dt,
+    unique_id: str,
+    start_time: dt,
+    end_time: dt,
 ) -> None:
     """Writes output metadata to plane processing.json
 
     Parameters
     ----------
     metadata: dict
         parameters from suite2p motion correction
-    input_fp: str
-        path to data input
+    raw_movie: str
```

Comment:
argument name is input_fp, not raw_movie

Reply:
and the input is not a raw movie but dF/F traces

Reply:
fixed to reflect the the generic var name.